kms-message/src/kms_kmip_tag_type_private.h

```diff
@@ -17,303 +17,325 @@
-   KMIP_TAG_CertificateIssuerO = 0x420115,
-   KMIP_TAG_CertificateIssuerOU = 0x420116,
-   KMIP_TAG_CertificateIssuerEmail = 0x420117,
-   KMIP_TAG_CertificateIssuerC = 0x420118,
-   KMIP_TAG_CertificateIssuerST = 0x420119,
-   KMIP_TAG_CertificateIssuerL = 0x42011A,
-   KMIP_TAG_CertificateIssuerUID = 0x42011B,
-   KMIP_TAG_CertificateIssuerSerialNumber = 0x42011C,
-   KMIP_TAG_CertificateIssuerTitle = 0x42011D,
-   KMIP_TAG_CertificateIssuerDC = 0x42011E,
-   KMIP_TAG_CertificateIssuerDNQualifier = 0x42011F,
-   KMIP_TAG_Sensitive = 0x420120,
-   KMIP_TAG_AlwaysSensitive = 0x420121,
-   KMIP_TAG_Extractable = 0x420122,
-   KMIP_TAG_NeverExtractable = 0x420123,
-   KMIP_TAG_ReplaceExisting = 0x420124
-} kmip_tag_type_t;
+#include "kms_message/kms_message_defines.h"
+
+#define KMS_XMACRO                                                                          \
+   KMS_X (ActivationDate, 0x420001)                                                         \
+   KMS_X (ApplicationData, 0x420002)                                                        \
+   KMS_X (ApplicationNamespace, 0x420003)                                                   \
+   KMS_X (ApplicationSpecificInformation, 0x420004)                                         \
+   KMS_X (ArchiveDate, 0x420005)                                                            \
+   KMS_X (AsynchronousCorrelationValue, 0x420006)                                           \
+   KMS_X (AsynchronousIndicator, 0x420007)                                                  \
+   KMS_X (Attribute, 0x420008)                                                              \
+   KMS_X (AttributeIndex, 0x420009)                                                         \
+   KMS_X (AttributeName, 0x42000A)                                                          \
+   KMS_X (AttributeValue, 0x42000B)                                                         \
+   KMS_X (Authentication, 0x42000C)                                                         \
+   KMS_X (BatchCount, 0x42000D)                                                             \
+   KMS_X (BatchErrorContinuationOption, 0x42000E)                                           \
+   KMS_X (BatchItem, 0x42000F)                                                              \
+   KMS_X (BatchOrderOption, 0x420010)                                                       \
+   KMS_X (BlockCipherMode, 0x420011)                                                        \
+   KMS_X (CancellationResult, 0x420012)                                                     \
+   KMS_X (Certificate, 0x420013)                                                            \
+   KMS_X (CertificateIdentifier, 0x420014)              /* deprecated as of version 1.1 */  \
+   KMS_X (CertificateIssuer, 0x420015)                  /* deprecated as of version 1.1 */  \
+   KMS_X (CertificateIssuerAlternativeName, 0x420016)   /* deprecated as of version 1.1 */  \
```

Comment:
This file requires ClangFormat. If we want to preserve current format of `KMS_XMACRO`, I recommend adding guards:
```cpp
// clang-format off
#define KMS_XMACRO                                                                          \
   KMS_X (ActivationDate, 0x420001)                                                         \
   /* ... */                                                                                \
   KMS_X (NeverExtractable, 0x420123)                                                       \
   KMS_X_LAST (ReplaceExisting, 0x420124)
// clang-format on
```